api/src/utils/mailer-templates.ts

```diff
@@ -1,15 +1,29 @@
-import { IUser, IPitch, IPitchAggregate } from 'ssw-common';
-import User from '../models/user';
+import { IUser, ITeam, IPitchAggregate } from 'ssw-common';
+import Team from '../models/team';
 
 export interface EmailMessage {
   to: string;
   from: string;
+  cc?: string;
   subject: string;
   html: string;
 }
+
+const buildEmailQuery = (
+  to: string,
+  subject: string,
+  html: string,
+): EmailMessage => ({
+  to: to,
+  from: process.env.EMAIL_USERNAME,
+  subject: subject,
+  html: html,
+});
+
 type userInfo = { name: string; email: string };
 type teamsUsers = Record<string, userInfo[]>;
-
+const getPitchDocLink = (): string =>
+  'https://south-side-weekly-dpoizlx6i-hack4impact1.vercel.app/pitches';
```

Comment:
I realized that this won't work because this deployment URL changes, we need to ask around to see if this is an error on us or if this is a vercel thing to change the production URL at every new push to main.